junit-platform-commons-java-9/src/test/java/org/junit/platform/commons/util/ModuleUtilsTests.java

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2015-2019 the original author or authors.
- *
- * All rights reserved. This program and the accompanying materials are
- * made available under the terms of the Eclipse Public License v2.0 which
- * accompanies this distribution and is available at
- *
- * https://www.eclipse.org/legal/epl-v20.html
- */
-
-package org.junit.platform.commons.util;
-
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import java.lang.module.ModuleDescriptor;
-import java.util.List;
-import java.util.Set;
-
-import org.junit.jupiter.api.Test;
-
-/**
- * Unit tests for {@link ModuleUtils}.
- *
- * @since 1.1
- */
-class ModuleUtilsTests {
```

Comment:
:x: Where did this test go?

Reply:
't was lost in translation. Will restore it.